src/aap_eda/settings/default.py

```diff
@@ -562,6 +563,7 @@ def get_rq_queues() -> dict:
     "PREPROCESSING_HOOKS": [
         "aap_eda.api.openapi.preprocess_filter_api_routes"
     ],
+    "GENERIC_ADDITIONAL_PROPERTIES": "bool",
```

Comment:
@Dostonbek1 Since this is going to change the generated  should we we change the SPECTACULAR_SETTINGS.VERSION to be the same as the EDA SERVER Version or will that break the generated OpenAPI client.

Reply:
We can do this in a separate PR and set the version to be 
```
VERSION: get_eda_version(),
```